lib/core/elastic_connector_actions.rb

```diff
@@ -159,13 +159,15 @@ def claim_job(connector_id)
         )
 
         body = {
-          :connector_id => connector_id,
           :status => Connectors::SyncStatus::IN_PROGRESS,
           :worker_hostname => Socket.gethostname,
           :created_at => Time.now,
           :started_at => Time.now,
           :last_seen => Time.now,
-          :filtering => convert_connector_filtering_to_job_filtering(connector_record.dig('_source', 'filtering'))
+          :connector => {
```

Comment:
I just made it to adopt the new mapping, but didn't add other connector data like `configuration`, `service_type`, etc..

This will be covered by @vidok ,  and this `claim` method will be split into an `enqueue` method and `claim` method.